examples/covariance/plot_outlier_detection.py

```diff
@@ -61,12 +66,16 @@
     # Add outliers
     X = np.r_[X, np.random.uniform(low=-6, high=6, size=(n_outliers, 2))]
 
-    # Fit the model with the One-Class SVM
+    # Fit the model
     plt.figure(figsize=(10, 5))
     for i, (clf_name, clf) in enumerate(classifiers.items()):
         # fit the data and tag outliers
-        clf.fit(X)
-        y_pred = clf.decision_function(X).ravel()
+
+        if clf_name=="Local Outlier Factor":
+            y_pred = -clf.fit_predict(X).ravel()
+        else:
+            clf.fit(X)
+            y_pred = clf.decision_function(X).ravel()
```

Comment:
this should be consistent between all estimators. LOF should not be a special case.